app/util/smtp.py

```diff
@@ -0,0 +1,39 @@
+import smtplib
+
+
+class SMTP():
```

Comment:
Is there any particular reason you decided to use old style class?

Reply:
there was typo, i fixed it